subsys/bluetooth/host/hci_core.h

```diff
@@ -481,3 +481,7 @@ void bt_hci_synchronous_conn_complete(struct net_buf *buf);
 
 void bt_hci_le_df_connection_iq_report(struct net_buf *buf);
 void bt_hci_le_df_cte_req_failed(struct net_buf *buf);
+
+void bt_hci_le_transmit_power_report(struct net_buf *buf);
+
+void bt_hci_le_path_loss_report(struct net_buf *buf);
```

Comment:
Maybe there's a precedent for this, but is `hci_core.h` really the right header file for functions defined in `conn.c`? Wouldn't `conn_internal.h` be more appropriate?

Reply:
It is needed in `hci_core `but defined in `conn ` to be able to access connections cbs. Maybe better me to define some `notify_x_cb` functions in `conn` (prototypes would be in `conn_internal.h`) and implement actual event functions in  `hci_core` like an example of   `le_remote_feat_complete() -> notify_remote_info()`